### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by MarioRubio81**

### CODE REVIEW

File: staff/mario-rubio/challenges/tic-tac-toe-app/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TIC-TAC-TOE</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="main.js"></script>
+</head>
+<body>
+    <div class="name">TIC-TAC-TOU</div>
```
Comment (nicolas-postigo) [inline staff/mario-rubio/challenges/tic-tac-toe-app/index.html:11]: tic-tac-toc

File: staff/mario-rubio/challenges/tic-tac-toe-app/main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+
+function TurnO() {
+    var PlayerO = document.write("Tu turno, Jugador 1!!!")
```
Comment (betoDavila86) [inline staff/mario-rubio/challenges/tic-tac-toe-app/main.js:4]: Don't forget the semicolons, mate ;)

**Review by betoDavila86 — CHANGES_REQUESTED** [review]

File: staff/mario-rubio/challenges/tic-tac-toe-app/index.html (hunk 1 above)
Comment (Aiidamontero) [reply staff/mario-rubio/challenges/tic-tac-toe-app/index.html:11]: tic-tac-toe

File: staff/mario-rubio/challenges/tic-tac-toe-app/main.js (hunk 2 above)
Comment (alexeh) [reply staff/mario-rubio/challenges/tic-tac-toe-app/main.js:4]: According to convention, function names should be written in camelCase!

File: staff/mario-rubio/challenges/calculator-app/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <script src="main.js"></script>
+
+</head>
+<body>
+    <div class="calculadora">
+        <section class="pantalla"> 
+            PANTALLA
+        </section>
+        <section class="fila">
+            <li class="numero">7</li><li class="numero">8</li><li class="numero">9</li><li class="dividir">/</li>
```
Comment (manuelbarzi) [inline staff/mario-rubio/challenges/calculator-app/index.html:17]: li should  only be used inside ul

File: staff/mario-rubio/challenges/tic-tac-toe-app/main.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+
+function TurnO() {
+    var PlayerO = document.write("Tu turno, Jugador 1!!!") 
+    return PlayerO
+}
+TurnO()
+
+function TurnX() {
```
Comment (alexeh) [inline staff/mario-rubio/challenges/tic-tac-toe-app/main.js:9]: Same here, try to writte names in camelCase as: oneName, nameTwo....

**Review by simonbesteiro — CHANGES_REQUESTED** [review]
Rename files inside tictactoe/imagenes properly

File: staff/mario-rubio/challenges/tic-tac-toe-app/main.js (hunk 2 above)
Comment (MarioRubio81) [reply staff/mario-rubio/challenges/tic-tac-toe-app/main.js:4]: Ok.  Continue working. thank you!

**Pull request closed without merging**